### ISSUE TITLE
AllowDuplicates = false not working as expected in overlapping sections

### ISSUE THREAD

**Issue by xozion** [doadin/Baggins#84]
Hey all,

i encountered an issue when using sections that overlap item categories, resulting in duplicate item displays, where there should be none. I want to have some sections to act as display for a big (parent)category and some other sections to filter out specific sub-categories.

### Reproduction steps
WowRetail 10.0.2 (german language) / Baggins 4.6.6
1. Create a bag
2. Create two sections: CraftItems, CookItems
3. Set section CraftItemsto: AllowDuplicates=false, Categoy: crafting material (priority 1)
4. Set section CookItems to: AllowDuplicates=false, Category: cooking (priority 2)

### Actual behaviour
Section "CraftItems" does display all crafting material related items, including cooking-stuff. At the same time section "CookItems" displays cooking-stuff items aswell resulting in duplicate item entries for all cook related items.
![image](https://user-images.githubusercontent.com/11751331/205281592-98370e42-970e-4954-b2b6-d4e02e018077.png)

### Expected behaviour
Cooking items are only displayed in section "CookItems" and NOT inside section "CraftItems" aswell.

Since section CookItems is set higher priority than section CraftItems, i expect it to have precedence when displaying cooking items and filtering them from the former section (as long as AllowDuplicates isnt set to true).

Is this a bug or did i configure something wrong?

**Comment by doadin** [comment]
In baggins config > Items. There is a dropdown menu for "Hide Duplicate Items" what do you have that set to?

**Comment by xozion** [comment]
Thanks for the hint, I just checked the setting, it was set to disabled. Didn't see it so far.
I changed it to bag/global, and the duplicate items inside the parent-category section disappeared (they're now only shown in the sub-category section with the high priority).

This is exactly what i wanted. So it basically was user-error on my side.
Thanks alot for the follow up, this fixed the issue! Feel free to close the ticket :)
